backend/app/adapter/sqldb/short_link_integration_test.go

```diff
@@ -724,6 +724,90 @@ func TestShortLinkSql_GetShortLinkByAliases(t *testing.T) {
 	}
 }
 
+func TestShortLinkSql_DeleteShortLink(t *testing.T) {
+	var (
+		twoYearsAgo = mustParseTime(t, "2018-05-01T08:02:16-07:00")
+		now         = mustParseTime(t, "2020-05-01T08:02:16-07:00")
+
+		longLink    = "https://short-d.com/"
+		customAlias = "short_is_great"
+	)
+
+	testCases := []struct {
+		name      string
+		tableRows []shortLinkTableRow
+		input     entity.ShortLinkInput
+		hasErr    bool
+	}{
+		{
+			name: "delete exisiting shortlink",
+			tableRows: []shortLinkTableRow{
+				{
+					alias:              "short_is_great",
+					longLink:           "https://short-d.com",
+					createdAt:          &twoYearsAgo,
+					expireAt:           &now,
+					ogTitle:            nil,
+					ogDescription:      nil,
+					ogImageURL:         nil,
+					twitterTitle:       nil,
+					twitterDescription: nil,
+					twitterImageURL:    nil,
+				},
+			},
+			input: entity.ShortLinkInput{
+				LongLink:    &longLink,
```

Comment:
```suggestion
				LongLink:    ptr.String("https://short-d.com/"),
```
Use `ptr.String` to directly inline the strings into the test case. Like this:

Reply:
Sweet didn't know that this was in use :100: